zone/bot_command.cpp

```diff
@@ -1743,14 +1743,57 @@ namespace MyBots
 		if (!bot_owner)
 			return;
 
-		if (!bot_owner->GetGroup())
+		if (!bot_owner->GetGroup() && !bot_owner->GetRaid())
 			return;
 
-		std::list<Bot*> group_list;
-		bot_owner->GetGroup()->GetBotList(group_list);
-		for (auto member_iter : group_list) {
-			if (IsMyBot(bot_owner, member_iter))
-				sbl.push_back(member_iter);
+		if (bot_owner->IsRaidGrouped()) {
+			Raid* raid = bot_owner->GetRaid();
+			if (!raid)
+				return;
+			uint32 raid_group = raid->GetGroup(bot_owner);
+			if (raid_group >= MAX_RAID_GROUPS) {
+				return;
+			}
+			for (int x = 0; x < MAX_RAID_MEMBERS; x++) {
+				if (raid->members[x].member && entity_list.IsMobInZone(raid->members[x].member) && raid->members[x].group_number == raid_group) {
+					if (!MyBots::IsMyBot(bot_owner, raid->members[x].member->CastToBot()))
+						continue;
+					sbl.push_back(raid->members[x].member->CastToBot());
+				}
+			}
+		}
+		else {
+			std::list<Bot*> group_list;
+			bot_owner->GetGroup()->GetBotList(group_list);
+			for (auto member_iter : group_list) {
+				if (IsMyBot(bot_owner, member_iter))
+					sbl.push_back(member_iter);
+			}
+		}
+
+		if (!clear_list)
+			UniquifySBL(sbl);
+	}
+
+	static void PopulateSBL_ByMyRaidBots(Client* bot_owner, std::list<Bot*>& sbl, bool clear_list = true) {
+		if (clear_list)
+			sbl.clear();
+		if (!bot_owner)
+			return;
+
+		if (!bot_owner->GetRaid())
+			return;
+
+		Raid* raid = bot_owner->GetRaid();
+		if (!raid)
+			return;
+
+		for (int x = 0; x < MAX_RAID_MEMBERS; x++) {
```

Comment:
Can probably do
```cpp
for (const auto& m : raid->members) {
    if (m.member) {
```

Don't think we have to check if mob is in zone if we have a valid pointer.